### PR TITLE
109 connect the forms on our webpages to hit our api accordingly

### PR DESCRIPTION
**Pull request by XxXgamerguyXxX**
# Pull request

**Resolves:** #<<109>>

### CODE REVIEW

**Comment by asmitakarandikar** [comment]
The text box that displays results is showing up on Safari, but it is not showing up on Chrome.

**Review by pelavarthi — DISMISSED** [review]
Looks god

File: darkmoon/api/templates/credit.html
Comment (DoodlePooge) [inline darkmoon/api/templates/credit.html:1]: Remove this from this PR, this is not for this ticket.

File: package-lock.json
Comment (DoodlePooge) [inline package-lock.json:1]: do we need this?

**Review by DoodlePooge — CHANGES_REQUESTED** [review]

**Review by DoodlePooge — APPROVED** [review]

**Pull request merged**